src/workerd/api/BUILD.bazel

```diff
@@ -775,3 +775,21 @@ wd_test(
     args = ["--experimental"],
     data = ["tests/messageport-test.js"],
 )
+
+js_binary(
+    name = "starttls-try-server",
+    entry_point = "tests/starttls-try-server.js",
+)
+
+wd_test(
+    src = "tests/starttls-try-test.wd-test",
```

Comment:
we have a pattern of `nodejs-server` suffix for the sidecar and `nodejs-test` suffix for the test. Can you rename the files? This helps running the test via `just node-test http-server` for example